### PR TITLE
add community topics

### PR DESCRIPTION
**Pull request by Connoropolous**
so this PR makes it so that if you're interested in the TOTAL number of posts for a topic, look at the Topic itself, and grab `postsTotal` off it, but if you're interested in the total number of posts for a topic within a community, grab it off of the CommunityTopic using `postsTotal`

### CODE REVIEW

**Comment by levity** [comment]
LGTM

File: api/models/Tag.js [hunk 1]
```diff
@@ -294,6 +294,24 @@ module.exports = bookshelf.Model.extend({
     })
   },
 
+  taggedPostCount: tagId => {
+    return bookshelf.knex('posts_tags')
```
Comment (Connoropolous) [inline api/models/Tag.js]: @levity should we be expanding this query, and another similar one in CommunityTag to filter for only `active` posts?

Comment (levity) [inline api/models/Tag.js]: yes

Comment (Connoropolous) [inline api/models/Tag.js]: K, will add that

**Comment by Connoropolous** [comment]
@levity k, fixed that, and added tests

**Pull request merged**